tests/test_algorithm.py

```diff
@@ -1095,10 +1096,63 @@ class TestPositions(WithLogger,
     START_DATE = pd.Timestamp('2006-01-03', tz='utc')
     END_DATE = pd.Timestamp('2006-01-06', tz='utc')
 
-    sids = ASSET_FINDER_EQUITY_SIDS = [1, 133]
+    equity_sids = ASSET_FINDER_EQUITY_SIDS = (1, 133)
```

Comment:
Nope, I had just left this here since it was defined before. I think I'll just delete it though as it doesn't really have a significant use other than as a convenient reference.